Cpp.Build.props

```diff
@@ -84,7 +84,7 @@
 
   <!-- Global props -->
   <PropertyGroup Label="Globals" Condition="'$(OverrideWindowsTargetPlatformVersion)'!='True'">
-    <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
```

Comment:
we should also change it other places, e.g. readme/installer

Reply:
done